src/open_inwoner/cms/plugins/views.py

```diff
@@ -112,6 +112,7 @@ def get(self, request: HttpRequest, plugin_id: int) -> HttpResponse:
                     # for 'formulieren':
                     # "naam" from API -> "description" for web component
                     "description": zaak.get("naam", zaak.get("description", "")),
+                    "is_formulier": zaak.get("type_aanvraag") == "Formulier",
```

Comment:
Use the constant:

Top of file:
`from open_inwoner.openzaak.constants import TypeAanvraag`

Then:
```suggestion
                    "is_formulier": zaak.get("type_aanvraag") == TypeAanvraag.FORMULIER.value,
```